### PR TITLE
Implemented Cron job authentication and entity diff checking

### PR DESCRIPTION
**Pull request by Ayaykos**
LocationDiff and MealDiff include any updates to the entities taken from the MDining API that need to be checked.

### CODE REVIEW

File: flask/requirements.txt [hunk 1]
```diff
@@ -1,3 +0,0 @@
-Flask==1.0.3
```
Comment (jonespm) [inline flask/requirements.txt:1]: This requirements file shouldn't have been deleted

File: flask/temp_comparator.txt [hunk 2]
```diff
@@ -0,0 +1,119 @@
+Baits EM MGifts
```
Comment (jonespm) [inline flask/temp_comparator.txt:1]: It looks like this file is just temp? It probably should be either part of gitignore or just use a tempfile https://docs.python.org/3.6/library/tempfile.html

**Review by jonespm — CHANGES_REQUESTED** [review]
The code in this looks good but there's a few comments here. Also you should create a branch and do a PR from that branch as mentioned in the first step on https://guides.github.com/introduction/flow/

This process keeps your master branch up-to-date in the fork up-to-date. https://help.github.com/en/articles/syncing-a-fork

File: flask/requirements.txt (hunk 1 above)
Comment (Ayaykos) [reply flask/requirements.txt:1]: Sorry, not sure how that happened.

File: flask/temp_comparator.txt (hunk 2 above)
Comment (Ayaykos) [reply flask/temp_comparator.txt:1]: It's not exactly a regular temp file that can be ignored—the way that the diff works is that it takes in two files to compare. One file is the regular Meal/LocationMain file we already have, but the other we need to create and insert in the updated Meals/Location nightly to compare with the preexisting Meal/LocationMain. Obviously there are methods that don't need this but it makes the diff process easier to work with and log changes.

**Comment by jonespm** [comment]
Closing as I believe this was already merged as part of #10

**Pull request closed without merging**